pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1774,6 +1774,10 @@
     dependencies = with self; [ vimwiki fzf-vim ];
   };
 
+  windows-nvim = super.windows-nvim.overrideAttrs {
+    dependencies = with self; [ luajitPackages.middleclass animation-nvim ];
```

Comment:
please dont hardcode the lua interpreter (`luajitPackages`) and use lua.pkgs instead. One can build neovim with PUC lua.

Reply:
ahh okay cool i wasn't sure which one to use :sweat: thanks